mup_scripts/mutransfer_lr/mup/run_openwebtext.sh

```diff
@@ -0,0 +1,59 @@
+# Single-GPU Launching
+LAUNCHER=python
+
+# Multi-GPU Launching (single node)
+#GPU=2
+#LAUNCHER=torchrun --standalone --nproc_per_node=$GPU
+
+for width in 256 512 1024 2048
+do
+    for lr in 0.0009765625 0.00048828125 0.000244140625 0.0001220703125 0.00006103515625
+    do
+        for seed in 1 2 3
+        do
+            head_size=64
+            n_heads=$((width / head_size))
+            mup_base_width=256
+            mup_width_multiplier=$(echo "scale=8; $width/$mup_base_width" | bc -l)
+            out_dir="mutransfer_lr/mup/out/width${width}_depth2_seed${seed}_lr${lr}"
+            $LAUNCHER train.py \
+                --out_dir=$out_dir \
+                --eval_interval=1 \
+                --log_interval=1 \
+                --eval_iters=1 \
+                --eval_only=False \
+                --skip_val_loss=True \
+                --always_save_checkpoint=False \
+                --never_save_checkpoint=True \
+                --init_from='scratch' \
+                --wandb_log=False \
+                --csv_log=True \
+                --dataset='openwebtext' \
+                --gradient_accumulation_steps=2\
+                --batch_size=4 \
+                --block_size=1024 \
+                --n_layer=2 \
+                --n_head=$n_heads \
+                --n_embd=$width \
+                --dropout=0.0 \
+                --bias=False \
+                --init_std=0.02 \
+                --learning_rate=$lr \
+                --max_iters=122 \
```

Comment:
@Quentin-Anthony would it be possible for you to rerun with 1000 iters and global batch size of 32? This would make things much less noisy. I only selected 122 iters because I was running on my macbook. Hopefully then we wouldn't need to plot "mean train loss" which is a little gross. Then this test has marginal value above the shakespeare_char test

Reply:
Can you also re-enable LR decay to 10x less than the maximum? I only chose constant because shakespeare_char was so small scale and short

Reply:
Running these now

Reply:
![image](https://github.com/user-attachments/assets/4b2accf1-dd52-44aa-be8b-9937dd3a91a6)